application-packages/DeployProcess_Stacker.json

```diff
@@ -0,0 +1,98 @@
+{
+    "processOffering": {
+        "process": {
+            "id": "Stacker",
+            "title": "Stacker",
+            "abstract": "",
+            "keywords": [],
+            "owsContext": {
+                "offering": {
+                    "code": "http://www.opengis.net/eoc/applicationContext/cwl",
+                    "content": {
+                        "href": "https://raw.githubusercontent.com/crim-ca/testbed14/master/application-packages/StackCreation-graph-json-remote.cwl"
+                    }
+                }
+            },
+            "inputs": [
+                {
+                    "id": "files",
+                    "title": "Input Image",
+                    "formats": [
+                        {
+                            "mimeType": "application/zip",
+                            "default": true
+                        }
+                    ],
+                    "minOccurs": 1,
+                    "maxOccurs": "unbounded",
+                    "additionalParameters": [
+                        {
+                            "role": "http://www.opengis.net/eoc/applicationContext/inputMetadata",
+                            "parameters": [
+                                {
+                                    "name": "EOImage",
+                                    "value": "true"
+                                }
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "id": "output_file_type",
+                    "title": "Type of Stacked Image",
+                    "formats": [
+                        {
+                            "mimeType": "text/plain",
+                            "default": true
+                        }
+                    ],
```

Comment:
I don't think there should be a "format" since its a "string" type?

Reply:
based on spacebel sample : https://github.com/spacebel/testbed14/blob/master/json-examples/DeployProcess_NDVI.json

Reply:
let's leave it as is since their demo has it, but feels off nonetheless since its LiteralData - String, so necessarily text/plain and not a File.

Reply:
Good point